pkg/broker/broker.go

```diff
@@ -708,6 +711,13 @@ func (a AnsibleBroker) Deprovision(
 		instance.Parameters = &params
 	}
 
+	// Override the lastRequestingUserKey value in the instance.Parameters
+	if instance.Parameters != nil {
+		params := *instance.Parameters
```

Comment:
So Parameters here is already a reference type, you don't have to deref and then copy the address back to Parameters. I'd update these on each to just be a one liner:

```
(*instance.Parameters)[lastRequestingUserKey] = getLastRequestingUser(userInfo)
```

https://play.golang.org/p/wG6632X68h7
```

Reply:
thanks, I will update